docs/tempo/website/configuration/multitenancy.md

```diff
@@ -19,8 +19,18 @@ exporters:
       x-scope-orgid: foo-bar-baz
 ```
 
+### Grafana 7.5.x and higher
+- Configure the Tempo datasource in Grafana to pass the tenant with the same header.
+```
+- name: Tempo-Multitenant
+  jsonData:
+    httpHeaderName1: 'X-Scope-OrgID'
+  secureJsonData:
+    httpHeaderValue1: 'foo-bar-baz'
+```
+
 ### Grafana 7.4.x
-- Configure the Tempo datasource in Grafana to pass the tenant as a bearer token. Yes, this is weird. It works b/c it is the only header that Jaeger can be configured to pass to its GRPC plugin.
+- Configure the Tempo datasource in Grafana to pass the tenant as a bearer token. This is necessary because it is the only header that Jaeger can be configured to pass to its GRPC plugin.
```

Comment:
lol, you didn't like my commentary about it being weird? :)

Reply:
Haha, it was a suggestion from Jita 😛 
https://github.com/grafana/tempo/pull/873#discussion_r688452564